src/main/java/core/basesyntax/service/filehandler/ReportGenerate.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service.filehandler;
+
+import core.basesyntax.db.CurrentData;
+import java.util.Map;
+
+public class ReportGenerate {
```

Comment:
```suggestion
public class ReportGenerator {
```

Reply:
not fixed, class names should be nouns, this is a verb